lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import ipaddress
+from ipaddress import ip_address
+import grpc
+import logging
+
+from magma.common.service_registry import ServiceRegistry
+from lte.protos.subscriberdb_pb2_grpc import SubscriberDBStub
+from magma.subscriberdb.sid import SIDUtils
+
+SERVICE_NAME = "subscriberdb"
+
+
+def get_subscriber_ip(sid: str) -> ip_address:
```

Comment:
you should encapsulate this inside a class that you inject into your new IPAllocator implementation. This will make the unit test setup a lot more straightforward because you can just control the return value of `get_subscriber_ip` instead of mocking out the subscriberdb stub at a global level

Reply:
done.